content/Phoenix for Rails developers - Bruno Vezoli.md

```diff
@@ -0,0 +1,347 @@
+# Phoenix for Rails developers: a practical example
+
+![Phoenix logo](images/phoenix-header.png)
+
+I started working on Ruby after an exploratory phase of many languages lead me to it. The syntax just captivated me, it was so expressive that it could be read like English and it had so many tools I could do things more productively than ever before. No more string helper functions or awkward loops to manipulate arrays, you could all those things out of the box and just focus on coding your solution.
+
+After a while I found Rails, and, although it was a bit difficult to get the hang of it, it felt right in with Ruby’s philosophy. I felt much more productive than with other tools and it came with batteries included. No hours of configurations or research for a slightly better library which just required you to do all the work, I could once more focus on what I wanted build and let Rails handle the details for me.
+
+Since that time I’ve tried many other web development frameworks in a wide variety of languages but I never felt at home with any of them. Even if many tried to replicate what Rails did there was always missing, whether it was because of the language or some decision where it diverged they always felt like mere copies to me.
+
```

Comment:
`Even if many tried to replicate what Rails did there was always missing`, `always something missing`?

Reply:
This is still missing, right?